### PR TITLE
move link style to its own class and apply it for Conitnue Reading button

### PR DESCRIPTION
**Pull request by klt39429**
Resolve issue #13 to add "Continue Button"
screen shot 2015-01-27 at 7 01 59 pm

### CODE REVIEW

**Comment by gfidente** [comment]
hi Khoi, thanks for helping.

So I think the idea is good but have a couple of comments:
1. I think this is meant to close issue #13 right? Can you mention that in the pull request as described here: https://github.com/blog/1506-closing-issues-via-pull-requests

2. I would prefer to see this implemented without changing the existing styles for the other elements, making the button to inherit what it needs and add what is missing based on position, what do you think about? I haven't tried myself though so I don't know how much hacking that would need.

**Comment by klt39429** [comment]
Hi Giulio, I would love to help and learn.

Are you referring to mixins? So we will have a class that defines the button style and the a tag from aside can include it? Then the a tag inside article can just use that class as is (no need to change base.html). For example:

```less
  .button_link {
    padding: 5px 15px 5px 15px;
    border: 3px solid @light-grey;
    .border-radius(15px);
    color: @red;
    text-decoration: none !important;
    font-size: 16px;
  }


aside {
  div#user_meta {
    ul {
      li {
        a {
          .button_link
        }
      }
```

**Comment by gfidente** [comment]
I am referring to mixins yes but you can't put that in the user_meta or it would only work for links in the left side of the page.

Instead you should customize the appearance of the 'a' items with 'button_link' applied into div#article_text

File: templates/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   </div>
   <div class="article_text">
     {{ article.summary }}
+    <a class="button_link" href="{{ SITEURL }}/{{ article.url }}">Continue reading →</a>
```
Comment (gfidente) [inline templates/index.html]: can you replace the arrow with &rarr; ?

File: static/css/style.less [hunk 2]
```diff
@@ -66,10 +76,7 @@ aside {
         line-height: 3em;
 
         a {
-          padding: 5px 15px 5px 15px;
-          border: 3px solid @light-grey;
-          .border-radius(15px);
-          color: @med-grey;
+          .button_link;
```
Comment (gfidente) [inline static/css/style.less]: I don't think we want this applied to aside_div a objects

File: static/css/style.less [hunk 3]
```diff
@@ -24,6 +24,16 @@ a {
   text-decoration: none;
 }
 
+.button_link {
```
Comment (gfidente) [inline static/css/style.less]: I think this should not be set globally but rather as a specialization of the a objects found within article_text

**Comment by gfidente** [comment]
closing for now because this didn't get updates in a while

**Pull request closed without merging**